### PR TITLE
Fix test scheduler

### PR DESCRIPTION
**Pull request by mirrorcoder**

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/105/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/106/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/107/

**Comment by miarmak** [comment]
Nice, thanks!
+1

File: tests/scheduler/scheduler.py [hunk 1]
```diff
@@ -30,16 +44,16 @@ def setUp(self):
...
 
         self.fake_process = mock.MagicMock()
-        self.process_patch = mockpatch.PatchObject(scheduler, 'Process',
+        self.process_patch = mockpatch.PatchObject(scheduler, 'WrapThreadTask',
                                                    new=self.fake_process)
         self.useFixture(self.process_patch)
-        self.fake_wrap_tt = mock.MagicMock()
-        self.fake_wrap_tt.__name__ = 'WrapThreadTask'
+        self.fake_wrap_tt = WrapThreadTaskFake()
+        # self.fake_wrap_tt.__name__ = 'WrapThreadTask'
```
Comment (roman-verchikov) [inline tests/scheduler/scheduler.py]: Debug leftover?

File: tests/scheduler/scheduler.py [hunk 2]
```diff
@@ -30,16 +44,16 @@ def setUp(self):
...
         self.wrap_tt_patch = mockpatch.PatchObject(scheduler, 'WrapThreadTask',
                                                    new=self.fake_wrap_tt)
         self.useFixture(self.wrap_tt_patch)
-        self.fake_base_task = mock.MagicMock()
-        self.fake_base_task.__name__ = 'BaseTask'
+        self.fake_base_task = BaseTaskFake()
+        # self.fake_base_task.__name__ = 'BaseTask'
```
Comment (roman-verchikov) [inline tests/scheduler/scheduler.py]: ditto

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/114/

**Comment by roman-verchikov** [comment]
+1.

**Comment by roman-verchikov** [comment]
We need someone else to approve to have this merged

**Comment by Settis** [comment]
+1 OK for me.

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/129/

**Pull request merged**